rules/manager.go

```diff
@@ -49,21 +45,23 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type"},
 	)
-	evalFailures = prometheus.NewCounter(
+	evalFailures = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: namespace,
 			Name:      "rule_evaluation_failures_total",
 			Help:      "The total number of rule evaluation failures.",
 		},
+		[]string{"rule_type"},
 	)
-	evalTotal = prometheus.NewCounter(
+	evalTotal = prometheus.NewCounterVec(
 		prometheus.CounterOpts{
 			Namespace: namespace,
 			Name:      "rule_evaluations_total",
 			Help:      "The total number of rule evaluations.",
 		},
+		[]string{"rule_type"},
```

Comment:
You should call `WithLabelValues` on all of these with both label values in the `init()` so that they're always exported.